integration-tests/src/test/java/oracle/weblogic/kubernetes/ItLivenessProbeCustomization.java

```diff
@@ -0,0 +1,487 @@
+    domainNamespace = namespaces.get(1);
+
+    // install and verify operator
+    installAndVerifyOperator(opNamespace, domainNamespace);
+
+    // create mii with additional livenessprobecustom script
+    String imageName = createAndVerifyDomainImage();
+
+    // create a basic model in image domain
+    createAndVerifyMiiDomain(imageName);
+  }
+
+  /**
+   * Verify the customization of liveness probe.
+   * Build model in image with liveness probe custom script for a 2 clusters domain.
+   * Enable "LIVENESS_PROBE_CUSTOM_SCRIPT" while creating domain CR.
+   * After domain is created copy the file named tempfile.txt into tested managed server pods in
+   * both clusters, which is used by custom script to trigger liveness probe.
+   * Verify the the container managed server pods in both clusters are restarted
+   */
+  @Test
+  @Order(1)
+  @DisplayName("Test customization of the liveness probe")
+  public void testLivenessProbe() {
+    Domain domain1 = assertDoesNotThrow(() -> getDomainCustomResource(domainUid, domainNamespace),
+        String.format("getDomainCustomResource failed with ApiException when tried to get domain %s in namespace %s",
+            domainUid, domainNamespace));
+    assertNotNull(domain1, "Got null domain resource");
+
+    // create temp file
+    String fileName = "tempFile";
+    String content = "This one line file is to test livenessProbe custom script";
+    File tempFile = assertDoesNotThrow(() -> createTempfile(fileName, content),
+        "Failed to create temp file");
+    logger.info("File created  {0}", tempFile);
+
+    for (int i = 1; i <= NUMBER_OF_CLUSTERS_MIIDOMAIN; i++) {
+      for (int j = 1; j <= replicaCount; j++) {
+        String managedServerPodName =
+            domainUid + "-" + CLUSTER_NAME_PREFIX + i + "-" + MANAGED_SERVER_NAME_BASE + j;
+
+        // get the restart count of the container in pod before liveness probe restarts
+        final int beforeRestartCount =
+            assertDoesNotThrow(() -> getContainerRestartCount(domainNamespace, null,
+            managedServerPodName, null),
+            String.format("Failed to get the restart count of the container from pod {0} in namespace {1}",
+                managedServerPodName, domainNamespace));
+        logger.info("For server {0} restart count before liveness probeis: {1}",
+            managedServerPodName, beforeRestartCount);
+
+        // copy script to pod
+        String destLocation = "/u01/tempFile.txt";
+        assertDoesNotThrow(() -> copyFileToPod(domainNamespace, managedServerPodName, "weblogic-server",
+            tempFile.toPath(), Paths.get(destLocation)),
+            String.format("Failed to copy file %s to pod %s in namespace %s",
+            tempFile, managedServerPodName, domainNamespace));
+        logger.info("File copied to Pod {0} in namespace {1}", managedServerPodName, domainNamespace);
+
+        try {
+          Thread.sleep(10 * 1000);
```

Comment:
Avoid hard-coded sleep.  Of course once we use Anil's suggestion, we do not need to copy temp file

Reply:
Removed